libraries/chain/database.cpp

```diff
@@ -1918,14 +1918,24 @@ namespace golos { namespace chain {
             calc_median(&chain_properties_19::max_delegated_vesting_interest_rate);
             calc_median(&chain_properties_19::custom_ops_bandwidth_multiplier);
 
+            const auto& dynamic_global_properties = get_dynamic_global_properties();
+
+            if (has_hardfork(STEEMIT_HARDFORK_0_19__952)) {
```

Comment:
It is better to set flag in get_dynamic_global_properties() on set min price.
And here do only checking of flag.